### PR TITLE
Updated READMEs

### PR DESCRIPTION
**Pull request by annenkov**

### CODE REVIEW

File: extraction/README.md [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
+Compiling Elm code:
+install the [Elm compiler] (https://guide.elm-lang.org/install/elm.html).
+Running Elm tests also requires `elm-explorations/test` package (see the required dependencies in
+`elm-extract/elm.json`)
+
+Some highlights from `theories`:
+
+
+* `theories/ExAst.v` -- An extension of the MetaCoq's certified erasure EAst data-structures with additional information about erased types. 
+* `theories/Erasure.v` -- An extension of the MetaCoq's certified erasure with erasure for types. Also implements erasure for global environments with extra typing information for global definitions.
```
Comment (jakobbotsch) [inline extraction/README.md:25]:
```suggestion
* `theories/Erasure.v` -- An extension of the MetaCoq's certified erasure with erasure for types and erasing only required dependencies. Also implements erasure for global environments with extra typing information for global definitions.
```

File: extraction/README.md [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+* `theories/ExTyping.v` -- Well-formedness of erased terms and erased global environments.
```
Comment (jakobbotsch) [inline extraction/README.md:26]: I think this file is not used anywhere as it is, we should be able to remove it (though I will probably use it for proving analysis correct in the future).

File: extraction/README.md [hunk 3]
```diff
@@ -1,7 +1,50 @@
...
+* `theories/ErasureCorrectness.v` -- Correctness lemmas for definition from `Erasure.v`.
```
Comment (jakobbotsch) [inline extraction/README.md:27]:
```suggestion
* `theories/ErasureCorrectness.v` -- Correctness lemmas for definitions from `Erasure.v`, proving that our erasure produces a well-formed erased environment.
```

File: extraction/README.md [hunk 4]
```diff
@@ -1,7 +1,50 @@
...
+* `theories/MetaCoqErasureCorrectnessStrong.v` -- A strengthened version of the correctness proof from MetaCoq (erasing dependencies only, not the full environment).
+* `theories/Extraction.v` -- High-level interface to extraction. Provides different pipelines for doing extraction with different trusted computing bases.
+* `theories/ExtractionCorrectness.v` -- Top-level correctness theorem relating the stages.
+* `theories/Optimize.v` -- Optimisations (dead argument elimination, logical parameters elimination) on `λ□` terms.
+* `theories/OptimizeCorrectness.v` -- Optimsations correctness.
```
Comment (jakobbotsch) [inline extraction/README.md:32]:
```suggestion
* `theories/OptimizeCorrectness.v` -- Correctness of optimisation (dearging).
```

File: extraction/README.md [hunk 5]
```diff
@@ -1,7 +1,50 @@
...
+* `theories/CertifyingEta.v` -- A eta-expansion procedure which generated proofs of equality of between the result and the original.
+* `theories/LPretty.v` -- Pretty-printer for Liquidity from `λ□`.
+* `theories/Liquidity.v` -- A pretty printer that works directly on the deep embedding of `λsmart` language.
+* `theories/LiquidityExtract.v` - A high-level interface to Liquidity extraction.
+* `theories/MidlangExtract.v` -- A high-level interface to Midlang extraction, incudes a pretty-printed and some examples.
```
Comment (jakobbotsch) [inline extraction/README.md:37]:
```suggestion
* `theories/MidlangExtract.v` -- A high-level interface to Midlang extraction, incudes a pretty-printer.
```
I deleted the examples.

Comment (annenkov) [reply extraction/README.md:37]: I'll bring them back :) I think we need examples for Midlang extraction. I tried with the counter - it works just fine.

File: extraction/README.md (hunk 2 above)
Comment (annenkov) [reply extraction/README.md:26]: I think it's imported in `OptimizeCorrectness.v`

Comment (annenkov) [reply extraction/README.md:26]: We still can keep it, but remove from highlights

Comment (annenkov) [reply extraction/README.md:26]: Damn, I forgot how to suggest changes :) @jakobbotsch could you delete the theories/ExTyping.v bit?

Comment (jakobbotsch) [reply extraction/README.md:26]:
```suggestion
```

File: extraction/README.md (hunk 5 above)
Comment (jakobbotsch) [reply extraction/README.md:37]: I agree, but I think we should put them in examples, not in theories then.

Comment (annenkov) [reply extraction/README.md:37]: Ok, let's remove this stuff here, I'll put them in examples.

**Pull request merged**